pkg/sources/github/github.go

```diff
@@ -483,16 +483,14 @@ func (s *Source) getReposByOrg(ctx context.Context, apiClient *github.Client, or
 	var numRepos, numForks int
 	for {
 		someRepos, res, err := apiClient.Repositories.ListByOrg(ctx, org, opts)
-		if err == nil {
-			defer res.Body.Close()
-		}
+		res.Body.Close()
```

Comment:
suggestion: We might want to check err != nil here first because if err != nil, res will be nil and will panic.

Reply:
You're totally right, not sure what I was thinking. Reverted to only close if err is nil like before, but not deferring the close